scss/components/_responsive-embed.scss

```diff
@@ -50,10 +49,14 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 @mixin foundation-responsive-embed {
   .responsive-embed, .flex-video {
-    @include responsive-embed;
+    @include responsive-embed( map-get($responsive-embed-ratios,default) );
 
-    &.widescreen {
-      padding-bottom: responsive-embed($responsive-embed-ratio-widescreen);
+    @each $name, $ratio in $responsive-embed-ratios {
+      @if $name != default {
+        &.#{$name} {
+          padding-bottom: responsive-embed($ratio);
+        }
+      }
```

Comment:
You are generating classes from the indexes of `$responsive-embed-ratios`. If it is a map intended to be customized by the user, I don't think it is safe.

For a lower specificity and to secure the `responsive-embed` modifiers inside a namespace, we could generate modifiers like `&-#{$name}` (so `.responsive-embed-panorama, .flex-video-panorama), or maybe as a generic utility with its own namespace (like `.ration-panorama`) ?

What do you think ?